include/ck/tensor_operation/gpu/grid/batchnorm_multiblock/gridwise_multiblock_welford_second_half_batchnorm_forward_final.hpp

```diff
@@ -268,8 +262,12 @@ struct GridwiseWelfordSecondHalfBatchNormForwardFinal
             welford_count_thread_buf(I) = 0;
         });
 
-        for(index_t reducedTiles = 0; reducedTiles < num_mean_var_count_k_block_tile_iteration;
-            ++reducedTiles)
+        constexpr index_t K_BlockTileSize_2 = KThreadClusterSize * 1;
```

Comment:
This may confuse with K_BlockTileSize, why not just use KThreadClusterSize directly?

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/771/commits/64139afdde025087e655439a15dfcb0b23371a59

Reply:
still the same

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/771/commits/95b78a773f437b349399bad4e7d77923f2e5ecf1